Src/Support/Google.Apis.Tests/Google.Apis.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
If you have `<IsPackable>False</IsPackable>` in test projects, you can call dotnet pack on the solution. I'm finding that to be really handy :)